reference/forms/types/options/choice_label.rst.inc

```diff
@@ -28,8 +28,8 @@ more control::
         },
     ]);
 
-This method is called for *each* choice, passing you the choice ``$value`` and the
-``$key`` from the choices array (``$index`` is related to `choice_value`_). This
+This method is called for *each* choice, passing you the ``$choiceValue`` and the
+``$key`` from the choices array (``$value`` is related to `choice_value`_). This
```

Comment:
the "``$value`` is related to `choice_value`_" part doesn't look right to me.

Reply:
what do you propose then?

Reply:
Maybe something like this:

```rst
This method is called for *each* choice, passing you the ``$choiceValue``
(related to `choice_value`_), the ``$key`` from the choices array and the
``$value``. This will give you:
```

Reply:
updated